labcontrol/gui/templates/normalization.html

```diff
@@ -192,22 +190,7 @@
   };
 
   $(document).ready(function(){
-    // Set the plate search table
-    var table = $('#searchPlateTable').DataTable(
-      {'ajax': {'url': '/plate_list', 'data': {'plate_type': JSON.stringify(['compressed gDNA']), 'only_quantified': true}},
-       'columnDefs': [
-         {'targets': -1, 'data': null,
-          'render': function(data, type, row, meta){
-            var plateId = data[0];
-            return "<button id='addBtnPlate" + plateId + "' class='btn btn-success btn-circle-small'><span class='glyphicon glyphicon-plus'></span></button>";
-          }},
-        ],
-        'order': [[0, "desc"]]});
-
-    // Add the function to the buttons that add the plate to the normalization process
-    $('#searchPlateTable tbody').on('click', 'button', function() {
-      addPlate(table.row( $(this).parents('tr') ).data()[0]);
-    });
+    setUpAddPlateModal(['compressed gDNA'], true);
```

Comment:
I may not have a strong enough mental model of the plate types: are there `compressed gDNA` plates that are not quantified? Just wondering if the `true` here is inferable / redundant.

Reply:
Yes, there can be `compressed gDNA` plates that are not quantified.  Compressed plates are created by compression.html, but there is an additional step necessary to record quantification values for them (parse_quantification.html).  For example, in my dev environment, I have 5 compressed gDNA plates but have only recorded quantifications for 2 of them--and therefore only those two can be normalized. Thus, I do not think that the `getOnlyQuantified` value of `true` here is inferable.